### ISSUE TITLE
Task 16 accuracy

### ISSUE THREAD

**Issue by EnricoBeltramo** [jimfleming/recurrent-entity-networks#18]
I have some issue to reach the expected accuracy on task 16. How should tune parameters in order to reach the expected accuracy?

**Comment by jimfleming** [comment]
Can you provide more information? What did you try, what version of the code, what do the logs say?

**Comment by EnricoBeltramo** [comment]
I cloned the actual trunk code and I tried different learning rate, but the accuracy is always around 0.5.
Here the final log using default traing rate:
INFO:tensorflow:global_step/sec: 6.85293
INFO:tensorflow:step = 34101, loss = 0.88134634 (14.653 sec)
INFO:tensorflow:global_step/sec: 6.91063
INFO:tensorflow:step = 34201, loss = 0.86507523 (14.409 sec)
INFO:tensorflow:global_step/sec: 6.64557
INFO:tensorflow:step = 34301, loss = 0.85566473 (15.049 sec)
INFO:tensorflow:Saving checkpoints for 34321 into checkpointstrain\model.ckpt.
Parameters: 49200
INFO:tensorflow:Starting evaluation at 2018-08-29-18:57:45
INFO:tensorflow:Graph was finalized.
INFO:tensorflow:Restoring parameters from checkpointstrain\model.ckpt-34321
INFO:tensorflow:Running local_init_op.
INFO:tensorflow:Done running local_init_op.
INFO:tensorflow:Finished evaluation at 2018-08-29-18:57:47
INFO:tensorflow:Saving dict for global step 34321: accuracy = 0.491, global_step = 34321, loss = 0.8779338
Stopping... Best step: 18721 with accuracy = 0.5040000081062317.
INFO:tensorflow:Loss for final step: 0.83799744.
Parameters: 49200

In this moment I'm using windows and tensorflow 1.8.0 (CPU). Should be that impact on the result?

**Comment by jimfleming** [comment]
I have not tested it at all on Windows. It might be an issue, I'm not sure. Some of the tasks do not converge consistently, which may also be the issue.
